karma.conf.js

```diff
@@ -52,7 +52,7 @@ module.exports = function(config) {
 		colors: true,
 		logLevel: config.LOG_INFO,
 
-		browsers: ['PhantomJS2'],
+		browsers: ['Chrome'],
```

Comment:
I don't understand why i can answer you everywhere except for the last couple messages.
![image](https://cloud.githubusercontent.com/assets/4950209/11699629/8efbb204-9ec5-11e5-9e4d-85d551a3965b.png)